modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -184,6 +184,26 @@ public void testAddCustomObjectField() throws Exception {
 						true
 					).build())));
 
+		// Localized object fields must be required
```

Comment:
I think you mean in the comment "Localized object fields must **not** be required". Is that it?